query-dsl/src/main/java/org/infinispan/query/dsl/QueryResult.java

```diff
@@ -0,0 +1,30 @@
+package org.infinispan.query.dsl;
```

Comment:
This package should ideally be ```org.infinispan.query```, and ```Query``` and ```QueryResult``` be moved to it, but this PR was already large